pydoctor/model.py

```diff
@@ -508,7 +508,7 @@ class System:
     def __init__(self, options: Optional[Values] = None):
         self.allobjects: Dict[str, Documentable] = {}
         self.rootobjects: List[Documentable] = []
-        self.warnings: Dict[str, List[Tuple[str, str]]] = {}
+        self.violations: Dict[str, List[Tuple[str, str]]] = {}
```

Comment:
maybe add a docstring with some info about what `violations` are.

I am still not sure how to define them, and what is the difference between `violations` and the remaining `_warnings`.

I would say that a warning like `cannot parse` should end up with a build failure...and not just be ignored.